source/DataBase/DataBaseManager.cpp

```diff
@@ -0,0 +1,32 @@
+
+#include "DataBaseManager.hpp"
+
+#include "QueryOperationBuilder.hpp"
+namespace dataBase {
+DataBaseManager::DataBaseManager(
+    std::shared_ptr<interface::IDataBaseConnection> dataBaseConnection,
+    std::shared_ptr<interface::IQueryOperationBuilder> queryOperationBuilder)
+    : dataBaseConnection_(std::move(dataBaseConnection)),
+      queryOperationBuilder_(std::move(queryOperationBuilder)) {}
+
+void DataBaseManager::executeOperation(
+    const common::DataBaseRequest &requestData) {
+  std::string query = queryOperationBuilder_->buildQuery(requestData);
```

Comment:
You're not handling the case where `query` is empty. If `buildQuery` returns an empty string, it might indicate an error, and you should handle this case appropriately.

Reply:
It is actually covered in line 18 :)